src/Silences/Types.elm

```diff
@@ -90,3 +32,20 @@ nullDuration =
 nullTime : Time
 nullTime =
     Utils.Date.fromTime 0
+
+
+type alias Silence =
+    { id : SilenceId
+    , createdBy : String
+    , comment : String
+    , startsAt : Time
+    , endsAt : Time
+    , duration : Duration
+    , updatedAt : Time
+    , matchers : List Matcher
+    , silencedAlertGroups : ApiData (List AlertGroup)
+    }
+
+
+type alias SilenceId =
```

Comment:
To me this adds unnecessary noise. The official Elm guide doesn't have this example: https://guide.elm-lang.org/types/type_aliases.html